code/FulpstationCode/Fulp_traitor_items/traitor_chaplain.dm

```diff
@@ -0,0 +1,250 @@
+	if(owner.mind.has_martialart(MARTIALART_STARTERSITH))
+		final_block_chance += FORCETRAINING_BLOCKCHANCE //+30% for having Sith training
+	SEND_SIGNAL(src, COMSIG_ITEM_HIT_REACT, args)
+	if(prob(final_block_chance))
+		spark_system.start()
+		playsound(src, pick('sound/weapons/blade1.ogg'), 75, TRUE)
+		owner.visible_message("<span class='danger'>[owner] deflects [attack_text] with [src]!</span>")
+		return TRUE
+	return FALSE
+
+
+/obj/item/melee/transforming/energy/sword/proc/spark_setup()
+	spark_system = new /datum/effect_system/spark_spread()
+	spark_system.set_up(5, 0, src)
+	spark_system.attach(src)
+
+
+
+/obj/item/book/granter/proc/sith_one_use(mob/user)
+	if(!oneuse)
+		return FALSE
+	var/datum/effect_system/spark_spread/spark_system = new /datum/effect_system/spark_spread()
+	spark_system.set_up(5, 0, src)
+	spark_system.attach(src)
+	spark_system.start()
+	if(isliving(user))
+		var/mob/living/L = user
+		L.electrocute_act(5,"Force Lightning",1,SHOCK_NOSTUN) //Snappy zappy.
+	to_chat(user,"<span class='warning'>In a flash of lightning, [src] suddenly crumbles to dust! No! The sacred texts!</span>")
+	playsound(src,'sound/magic/lightningbolt.ogg', 30, TRUE)
+	new /obj/effect/decal/cleanable/ash(get_turf(src))
+	qdel(src)
+
+
+/obj/effect/proc_holder/spell/targeted/force_lightning
+	name = "Force Lightning"
+	desc = "Release a blast of force lightning at a random target within 4 tiles. Arcs between targets, and decays in power with each arc."
+	charge_type = "recharge"
+	charge_max	= 10
+	clothes_req = FALSE
+	range = 4
+	cooldown_min = 10
+	selection_type = "view"
+	random_target = TRUE
+	invocation = null
+	invocation_type = null
+	var/static/mutable_appearance/halo
+	var/sound/Snd // so far only way i can think of to stop a sound, thank MSO for the idea.
+	var/bolt_flags = SHOCK_NOGLOVES | SHOCK_NOSTUN
+	var/max_bounces = 10
+	var/bolt_decay_per_bounce = 4 //How much damage does the bolt lose when it bounces.
+	var/min_damage = 5
+	var/max_damage = 20
+	var/stamina_damage_multiplier = 0.5
+	var/datum/effect_system/spark_spread/spark_system
+	var/list/existing_targets = list()
+
+	action_icon_state = "lightning"
+
+/obj/item/clothing/head/helmet/chaplain/sith
+	name = "sith hood"
+	desc = "It stinks of old man, ozone and the Dark Side."
+	icon_state = "crusader"
+	item_state = "crusader"
+	flags_cover = HEADCOVERSEYES
+	flags_inv = HIDEHAIR|HIDEFACE|HIDEEARS
+
+/obj/item/clothing/suit/armor/riot/chaplain/sith
+	name = "sith robes"
+	desc = "They stink of old man, ozone and the Dark Side."
+	icon_state = "crusader"
+	item_state = "crusader"
+
+
+/obj/effect/proc_holder/spell/targeted/force_lightning/proc/spark_setup()
+	spark_system = new /datum/effect_system/spark_spread()
+	spark_system.set_up(5, 0, src)
+
+
+/obj/effect/proc_holder/spell/targeted/force_lightning/cast(list/targets, mob/user = usr)
+	if(!can_cast(user))
+		return
+
+	if(iscarbon(user))

```

Comment:
check to see if they have at least one hand to shoot the lightning from

Reply:
Sure.